theme/snippets/footer.liquid

```diff
@@ -0,0 +1,24 @@
+
+  <footer class="Footer w100 serif-xl">
+    <p class="px1_25">
+      Contact us if you’d like to teach a workshop, propose an event, feature a product or get involved in any other way. You can also sign up for 
+      {% if shop.metafields.accentuate.newsletter %}
```

Comment:
Typically this text should be in the Language file, but is there a good way to do this when you have links within the text? Is it ok to leave this hardcoded? 
![image](https://user-images.githubusercontent.com/43737723/104762850-a7c6d200-572a-11eb-96ea-fe94f3d61773.png)

Reply:
Yeah, this seems like a fine approach here.